cli/tools/repl/mod.rs

```diff
@@ -58,9 +60,17 @@ async fn read_line_and_poll(
   }
 }
 
+fn read_script(script_path: String) -> Result<String, AnyError> {
+  let mut file = File::open(script_path)?;
+  let mut data = Vec::new();
+  file.read_to_end(&mut data)?;
+  Ok(String::from_utf8(data)?)
+}
```

Comment:
We should reuse facilities from `FileFetcher` to provide ability to load remote scripts as well

Reply:
This is great feedback, thank you. I will make these changes.